ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/ledger/sql/dao/HikariJdbcConnectionProvider.scala

```diff
@@ -46,6 +46,18 @@ class HikariJdbcConnectionProvider(
       connectionTimeout: FiniteDuration) = {
     val config = new HikariConfig
     config.setJdbcUrl(jdbcUrl)
+
+    //Typically you would pass the userid and password in the jdbc url but the url format is 
+    //different for DB2 (jdbc:db2://myhost:5021/mydb:user=dbadm;password=dbadm;) and these values
+    //are getting lost somewhere.
+    config.setUsername("username")
```

Comment:
Doesn't this break Postgres? Isn't this effectively hardcoding username/password?

Reply:
Yes, we'll need to understand what's going on with the JDBC URI.